tmpfiles.d/flatpak-extension-kiwix-usb-content.conf

```diff
@@ -0,0 +1,3 @@
+D /run/flatpak/extension/com.endlessm.encyclopedia.en.Content/x64_64 0755 - - -
+L /run/flatpak/extension/com.endlessm.encyclopedia.en.Content/x64_64/eos3 - - - - /run/kiwixhome/flatpak
```

Comment:
IDK, the way I have thought of this is making it a runtime extension, not overriding the runtime. Maybe it would be better to make it a runtime as you suggest, let's discuss on the next meeting.

Reply:
Ah, that path is for unmanaged extensions! Can't find real docs, but https://blog.tingping.se/2018/08/26/flatpak-host-extensions.html :)

Reply:
We also discussed that this means the files on the exFAT partition must be in the layout expected by the SDK. The layout appears to be:

```
/$SUBSCRIPTION_NAME
  /manifest.json
  /1.zim
```

Reply:
Per the discussion on today's meeting, I'm keeping the paths as-is here.